StaticAnalyzer/views/android/manifest_view.py

```diff
@@ -37,6 +43,8 @@ def run(request):
                 'file': 'AndroidManifest.xml',
                 'dat': manifest
             }
+            if api:
+                return context

```

Comment:
for API, it should be 
```
return {'file': 'AndroidManifest.xml', 'contents': manifest }
```
Then again the similar response structure should be followed for IPA or APPX as well